app/resources/api/rest/admin/routing/destination_resource.rb

```diff
@@ -39,6 +39,8 @@ class Api::Rest::Admin::Routing::DestinationResource < ::BaseResource
   ransack_filter :reverse_billing, type: :boolean
   ransack_filter :profit_control_mode_id, type: :number
   ransack_filter :rate_policy_id, type: :number
+  ransack_filter :rateplan_id, type: :number, column: :rate_group_rateplans_id
```

Comment:
`app/services/ransack_filter_builder.rb`
Please add `foreign_key: %w[eq not_eq in not_in]` into `RANSACK_TYPE_SUFIXES_DIC`

And use `type: : foreign_key` for these new filters